knowledge-base/bid-decay-mechanism.mdx

```diff
@@ -0,0 +1,48 @@
+
+When a provider decides to commit to the bid, they must issue a commitment and also assign a "commitment timestamp" to it. This commitment is then sent to the mev-commit chain for inclusion. The mev-commit chain will include the commitment into the chain, however the commitment will only be considered valid if the commitment timestamp supplied by the provider and the block timestamp where the commitment was included are "close enough", or more formally, are within some time interval $\Delta$.
+
+The parameter $\Delta$ is set to be $250$ms, and its goal is to account for the latency in message delivery among nodes and the discrepancy between the commitment and block timestamps. More precisely, for validation of a commitment it is required that $t_b - t_c \le \Delta$, where $t_c$ denotes the commitment timestamp and $t_b$ denotes the timestamp of the block where the commitment was included.
+
+## Computing the bid decay
+
+Once the commitment is included in the chain it can be publicly determined if the commitment is valid, and if so, what is the value of the decay of the commitment.
+
+The bid's value decreases linearly from the moment it is issued until its expiry deadline is reached. Below we show a diagram of how providers can use the timestamps to first compute the proportion to which a commitment has decayed. Once the proportion has been computed the value of the bid can be decayed accordingly.
+
+<Frame>
+  <img src="/images/bid-decay-proportion.png" />
+</Frame>
+
+For example, if the decay proportion is $1/2$ as in the example above, the bid's value decays by $50$%.
+
+
+## Analysis of the mechanism
+
+The security of the mechanism comes from the fact that the participating parties have conflicting incentives. In particular, we note that there is no incentive for the bidder to manipulate the choice of the bid’s timestamps; if the bidder includes a timestamp in the future, then the provider essentially gets less decay (i.e., the bidder pays more). If the bidder includes a timestamp that is too old, then the provider can choose to not commit as the decay is too high (then the bidder gets no commitment). Moreover, we note that this would have the exact same effect as the bidder changing the price of its bid prior to the submission of the bid. Similarly, the commitment timestamp set by the provider must not be too old, as otherwise, it won't be valid when included in the mev-commit chain.
+
+In many network-based systems, including our decay mechanism, faster connections often provide better opportunities, such as quicker access to information or response times. This situation is similar to high-frequency trading in financial markets, where traders with faster connections can outperform others. In our system, any advantage due to a faster connection is bounded by the actual latency a provider has to the mev-commit chain.
```

Comment:
The advantage would be bounded by the maximally allowed latency, not the one a given provider actually has.